crates/bevy_core_pipeline/src/tonemapping/mod.rs

```diff
@@ -54,19 +84,75 @@ pub struct TonemappingPipeline {
     texture_bind_group: BindGroupLayout,
 }
 
-#[derive(Copy, Clone, PartialEq, Eq, Hash)]
+#[derive(
+    Component,
+    Debug,
+    Hash,
+    Clone,
+    Copy,
+    Reflect,
+    Default,
+    ExtractComponent,
+    PartialEq,
+    Eq,
+    FromReflect,
+)]
+#[extract_component_filter(With<Camera>)]
+#[reflect(Component)]
+pub enum Tonemapping {
+    None,
+    /// Suffers from lots hue shifting, brights don't desaturate naturally.
+    Reinhard,
+    /// Old bevy default. Suffers from hue shifting, brights don't desaturate much at all.
+    ReinhardLuminance,
+    /// Bad
+    Aces,
+    /// Very Good
+    AgX,
+    /// Also good
+    SomewhatBoringDisplayTransform,
+    /// Very Good
+    TonyMcMapface,
+    /// Also good
+    #[default]
+    BlenderFilmic,
+}
```

Comment:
Please document these in a way that guides people to choose what they want. I think most of what we have been talking about kind of boils down to primarily desaturation in bright values, hue shifting, and sometimes we talk about contrast/looking blown out. These terms can serve as hints for what to look for in the example to see what difference they make in practice. It takes time and guidance for people to learn to see the differences. :)

Reply:
I don't think rating these or calling any "Bad" makes sense. It begs the question why it's even there if it's bad, and in this case, why it's a film industry/UE standard. Listing pros/cons/criticisms would be more helpful.

Reply:
Agreed. I've been planning on updating this. If you are referring to ACES, according to some colorists in the industry it's not actually standard in film, it's just marketed that way.